changelog/255.fixed.md

```diff
@@ -0,0 +1 @@
+Refactor Timestamp to use `whenever` instead of `pendulum` and extend Timestamp with add(), subtract(), and to_datetime(). Direct access to `obj` and `add_delta` have been deprecated and will be removed in a future version.
```

Comment:
I think we could split this up into two fragments where we also use the special one for things marked for deprecation.

Reply:
Good point, I forgot about the deprecated section